pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -197,6 +201,7 @@ public void registerTenant(String resourceGroupName, String tenantName) throws P
 
         // Associate this tenant name with the RG.
         this.tenantToRGsMap.put(tenantName, rg);
+        rgTenantRegisters.labels(resourceGroupName).inc();
```

Comment:
I would suggest you cache rgTenantRegisters.labels(resourceGroupName) so that you don't have to call it every single time to add a metric.  `labels()` allocates a new array list every time you call the method

Reply:
nvm

Reply:
`rgTenantRegisters.labels(resourceGroupName)` will be different for each `resourceGroupName` that calls into the above code. Are you suggesting that we maintain another structure to look up the appropriate `rgTenantRegisters.labels(resourceGroupName)` from the `resourceGroupName`, and use the result of that lookup in the above call?